EventAggregator.js

```diff
@@ -1,28 +1,43 @@
-var EventAggregator = function() {
+var EventAggregator = function(options) {
```

Comment:
@bschlintz I'm not grasping the reason for this. If you pass an instance ID to the constructor then all events using that EventAggregator instance will be using that event key prefix.  There's no way to use the same EventAggregator with a different key prefix.  To do that you would have to create a different EventAggregator instance, a which point you can have events with the same because they are on different EventAggregator instances....